content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -121,15 +124,28 @@ Google Calendar with Odoo.
    :alt: Click the Google sync button in Odoo Calendar to sync Google Calendar with Odoo.
 
 .. note::
-   The first time you sync your Google Calendar with Odoo, the page will redirect to your Google
-   Account. Click :guilabel:`OK` and :guilabel:`Allow` to authorize Odoo to access Google Calendar.
+   The first time syncing Google Calendar with Odoo, the page will redirect to the Google
+   Account. Select the :menuselection:`Email Account` that should have access -->
+   :menuselection:`Continue` (should the app be unverifed) --> :menuselection:`Continue` (to give
+   permission for the transfer of data)
```

Comment:
While synching Google Calendar with Odoo for the first time, the page will redirect to the Google
   Account. From there, select the :menuselection:`Email Account` that should have access -->
   :menuselection:`Continue` (should the app be unverifed) --> :menuselection:`Continue` (to give
   permission for the transfer of data).